src/flb_io.c

```diff
@@ -102,6 +102,15 @@ int flb_io_net_connect(struct flb_upstream_conn *u_conn,
                   u_conn->fd, u->tcp_host, u->tcp_port);
     }
 
+    /* set TCP keepalive and it's options */
+    if (u->net.tcp_keepalive != FLB_FALSE) {
```

Comment:
Please either use `if (u->net.tcp_keepalive == FLB_TRUE) {` or the preferred syntax according to the coding style `if (u->net.tcp_keepalive) {`.